tests/test_src/test_transaction/test_transaction.cpp

```diff
@@ -5,13 +5,15 @@
 #include "transaction/esdt.h"
 #include "transaction/signer.h"
 #include "transaction/transaction.h"
+#include "transaction/messagesigner.h"
 #include "wrappers/cryptosignwrapper.h"
 
 // Most tests for signing and transaction construction are adapted from one of the following sources:
 // ERD-JS: https://github.com/ElrondNetwork/elrond-sdk-erdjs/blob/bb926b029150d7c79f2b37308f4334f98a4cabf7/src/testutils/wallets.ts#L110
 //         https://github.com/ElrondNetwork/elrond-sdk-erdjs/blob/main/src/walletcore/users.spec.ts#L120
 // ERD-PY: https://github.com/ElrondNetwork/elrond-sdk-erdpy/blob/main/erdpy/tests/test_wallet.py
 // ERD-GO: https://github.com/ElrondNetwork/elrond-go/blob/master/examples/construction_test.go
+//         https://github.com/ElrondNetwork/elrond-go/blob/7d133422048b7e65f0a8730cfdd1fff9d51077e9/examples/messageSign_test.go#L87
```

Comment:
`main` instead of `7d13...`? Oh, but then the line might change in the future. Can stay as it is.